doc/Building_targets.md

```diff
@@ -46,13 +39,16 @@ block you from building. Remove them if you get the following ld error
 /usr/lib/gcc-cross/arm-linux-gnueabihf/4.8/../../../../arm-linux-gnueabihf/bin/ld: cannot find -lglib-2.0
 ```
 
-**NOTE 2:** In the specific case of Cervantes, Kindle & Kobo targets, while we make some effort to support these Linaro/Ubuntu TCs,
-they do *not* exactly target the proper devices. While your build will go fine, this may lead to runtime failure.
-As time goes by, and/or the more bleeding-edge your distro is, the greater the risk for mismatch gets.
-Thankfully, we have a distribution-agnostic solution for you: [koxtoolchain](https://github.com/koreader/koxtoolchain)!
-This will allow you to build the *exact* same TCs used to build the nightlies, thanks to the magic of [crosstool-ng](https://github.com/crosstool-ng/crosstool-ng). These are also included precompiled in the Docker images for the respective targets.
+##### e-Ink devices (e.g., Kindle, Kobo, Cervantes, reMarkable, PocketBook)
+
+**NOTE:** While, for some targets (specifically, Cervantes, Kindle & Kobo), we make *some* effort to support Linaro/Ubuntu TCs,
+they do *not* exactly target the proper devices. While your build *may* go fine, this will *probably* lead to runtime failure.  
```

Comment:
Not specific to this line, there's a lot of trailing whitespace hanging around?
```suggestion
they do *not* exactly target the proper devices. While your build *may* go fine, this will *probably* lead to runtime failure.
```

Reply:
That's to force a linebreak in MarkDown ;).

Reply:
(i.e., keep things in the same block, but with an explicit break between sentences to make the thing more readable).

Reply:
Not the biggest fan of what that looks like, but alright.